homeassistant/components/alarm_control_panel/egardia.py

```diff
@@ -98,15 +95,15 @@ def lookupstatusfromcode(self, statuscode):
         if self._rs_codes is not None:
             statuscode = str(statuscode).strip()
             for i in self._rs_codes:
-                val = str(self._rs_codes[i]).strip()
+                val = str(self._rs_codes[i][0]).strip()
```

Comment:
After the new config schema, we expect `self._rs_codes[i]` to return a list. Why only access the first item in the list?

Reply:
Also since this is a dict that is iterated, you can do that like so:
```py
for status_type, codes in self._rs_codes.items():
    ...
```
And please use another iteration variable than `i` for the key.